bom/runtime/pom.xml

```diff
@@ -224,6 +224,15 @@
                 <type>pom</type>
             </dependency>
 
+            <!-- Kogito dependencies, imported as a BOM -->
+            <dependency>
+                <groupId>org.kie.kogito</groupId>
+                <artifactId>kogito-bom</artifactId>
+                <version>${kogito.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
```

Comment:
I'm not sure it's worth it to import the Kogito BOM. Can't the users import both the Quarkus BOM and the Kogito BOM in their apps?

/cc @aloubyansky

Reply:
That is my sentiment as well, however I am not a maven black wizard :)

Reply:
Main reason is to avoid version mismatch different kogito version coming with quarkus and then different one selected by users

Reply:
Yeah, I see the point of it. My main interrogation is: should we add the BOM of all the technologies we use in our BOM. Kogito being a very specific component, I was thinking that maybe the user could include the BOM themselves.

I know @aloubyansky was a bit reluctant to add too many BOMs in ours.

Reply:
There is a supported extension for Kogito, so in this regard it's not different from other supported extensions that added their BOMs to the Quarkus one. So, based on that, I'd say lets add it.
However, this is something to think about, as our BOM will keep expanding a lot and rapidly. We may want to consider having a pure Quarkus-core BOM and then let users (and/or our tolling) import the rest separately. There will be different issues with that approach though.